pkg/deploy/registry/deploylog/rest.go

```diff
@@ -25,8 +26,12 @@ import (
 	deployutil "github.com/openshift/origin/pkg/deploy/util"
 )
 
-// defaultTimeout is the default time to wait for the logs of a deployment
-const defaultTimeout time.Duration = 20 * time.Second
+const (
+	// defaultTimeout is the default time to wait for the logs of a deployment.
+	defaultTimeout time.Duration = 20 * time.Second
```

Comment:
> what error code we return when this timeout is reached?

Depends on where do we reach it. If we timeout while waiting for the replication controller to be created, we return a notFound error because we were polling on that error. If we reach this while waiting for the replication controller to become running then we return a timeout error.

> can client retry getting the logs based on the return code?

It's possible. Do we want to do something for oc?